src/Keboola/GeocodingAugmentation/Augmentation.php

```diff
@@ -218,32 +214,29 @@ protected function setupProvider($httpAdapter, $config)
                 if (!isset($config['privateKey'])) {
                     throw new Exception("Provider {$config['provider']} needs 'privateKey' attribute configured");
                 }
-                return new GoogleMapsBusiness(
-                    $httpAdapter,
-                    $config['clientId'],
-                    $config['privateKey'],
-                    $locale,
-                    null,
-                    true
-                );
+                return GoogleMaps::business($httpClient, $config['clientId'], $config['privateKey']);
                 break;
             case 'bing_maps':
-                return new BingMaps($httpAdapter, $config['apiKey'], $locale);
+                return new BingMaps($httpClient, $config['apiKey']);
                 break;
             case 'yandex':
-                return new Yandex($httpAdapter, $locale);
+                return new Yandex($httpClient);
                 break;
             case 'map_quest':
-                return new MapQuest($httpAdapter, $config['apiKey'], $locale);
+                return new MapQuest($httpClient, $config['apiKey']);
                 break;
             case 'tomtom':
-                return new TomTom($httpAdapter, $config['apiKey'], $locale);
+                return new TomTom($httpClient, $config['apiKey']);
                 break;
             case 'opencage':
-                return new OpenCage($httpAdapter, $config['apiKey'], true, $locale);
+                return new OpenCage($httpClient, $config['apiKey']);
                 break;
-            case 'openstreetmap':
```

Comment:
Tohle vypadá na BC break. Nemohlo by to pořád akceptovat i `openstreetmap` a mapovat to na `nominatim`?

Reply:
Už to vidím že tohle pořešené je https://github.com/keboola/geocoding-augmentation/pull/10/files#diff-ad422aba7024f7229431aa93a489ce72R42